dmipy/core/modeling_framework.py

```diff
@@ -2167,7 +2172,8 @@ def __call__(self, acquisition_scheme, **kwargs):
         kwargs: keyword arguments to the model parameter values,
             Is internally given as **parameter_dictionary.
         """
-        A = self._construct_convolution_kernel(**kwargs)
+        A = self._construct_convolution_kernel(
```

Comment:
can you also please remove the `quantity` parameter definition as it is deprecated?

damn this code needs a clean up...

Reply:
my code that is, the MR is fine ;)